app/src/main/java/com/github/epfl/meili/profile/favoritepois/FavoritePoisActivity.kt

```diff
@@ -0,0 +1,97 @@
+package com.github.epfl.meili.profile.favoritepois
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.*
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.ViewModelProvider
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.github.epfl.meili.R
+import com.github.epfl.meili.database.FirestoreDatabase
+import com.github.epfl.meili.forum.ForumActivity
+import com.github.epfl.meili.home.Auth
+import com.github.epfl.meili.map.MapActivity
+import com.github.epfl.meili.models.PointOfInterest
+import com.github.epfl.meili.models.VisitedPointOfInterest
+import com.github.epfl.meili.util.MeiliViewModel
+import com.github.epfl.meili.util.TopSpacingItemDecoration
+import java.util.*
+
+
+class FavoritePoisActivity : AppCompatActivity() {
+    companion object {
+        private const val CARD_PADDING: Int = 30
+        private const val ACTIVITY_TITLE = "Favorite POIs"
+        private const val DB_PATH = "poi-history/%s/poi-history"
```

Comment:
Sorry, I forgot to mention it in the review but do you think we could change the path to something like `favourite-poi...`?